<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPlayerDialogue</class>
 <widget class="QDialog" name="EditPlayerDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edytuj gracza</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
       <item>
        <widget class="QComboBox" name="ComboBoxSelectPlayer"/>
       </item>
       <item>
        <widget class="QPushButton" name="BttnConfirmSelection">
         <property name="text">
          <string>WYBIERZ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="LabelAuthor">
         <property name="text">
          <string>Autor:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="TxtEditAuthor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LabelName">
         <property name="text">
          <string>Imię:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="TxtEditCharName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LabelLvl">
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="SpinBoxLvl">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LabelAttrubutes">
         <property name="text">
          <string>Atrybuty:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="ComboBoxAttributes"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="ComboboxAttributres2"/>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="CheckBoxMartialArmor">
         <property name="text">
          <string>Martial zbroja</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="CheckBoxMartialWeapon">
         <property name="text">
          <string>Martial broń</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="CheckBoxRange">
         <property name="text">
          <string>Zasięgowa broń</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="CheckBoxShield">
         <property name="text">
          <string>Tarcza</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="ComboBoxWeaponPreference"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="LabelPreference">
         <property name="text">
          <string>Preferencja:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="BttnDelPlayer">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditPlayerDialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditPlayerDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
